docs/http-api/persistent.md

```diff
@@ -104,7 +110,7 @@ Read [Reading Streams](README.md#reading-streams-and-events) for information on
 
 ## Acknowledgements
 
-Clients must acknowledge (or not acknowledge) messages in the competing consumer model. If the client fails to respond in the given timeout period, the message is retried. You should use the `rel` links in the feed for acknowledgements not bookmark URIs as they are subject to change in future versions.
+Clients must acknowledge (or not acknowledge) messages in the competing consumer model. If processing is successful, send an **Ack** (acknowledge) to the server to let it know that the message has been handled. If processing fails, then you can **Nack** (not acknowledge) the message and tell the server how to handle the failure. If the client fails to respond in the given timeout period, the message is retried. You should use the `rel` links in the feed for acknowledgements rather than bookmark URIs, as they are subject to change in future versions.
```

Comment:
```suggestion
Clients must acknowledge (or not acknowledge) messages in the competing consumer model. If processing is successful, send an **ack** (acknowledge) to the server to let it know that the message has been handled. If processing fails, then you can **nack** (not acknowledge) the message and tell the server how to handle the failure. If the client fails to respond in the given timeout period, the message is retried. You should use the `rel` links in the feed for acknowledgements rather than bookmark URIs, as they are subject to change in future versions.
```